app/util/get-version.js

```diff
@@ -10,8 +10,18 @@
 
 'use strict';
 
+var argv = require('mri')(process.argv);
```

Comment:
What do we want to fix here? I assume we need the nightly for both client builds and distro builds?

Where do we feed `--nightly` into the webpack configuration?

Reply:
It could make sense to clearly document where the utility gets its information from; potentially we could even consider to switch away from args for nightly builds to env variables, too, i.e. `BUILD_NIGHTLY=1`. As it stands this is rather confusing.

Reply:
This ensures that the version used in building is always the same throughout the build process. The utility is purposefully no longer configurable to prevent future mismatches.

`--nightly` is passed from CI to `npm build` here: https://github.com/camunda/camunda-modeler/blob/develop/.github/workflows/NIGHTLY.yml#L46

Let's switch to ENV variables instead, makes it easier to understand what is happening 👍

Reply:
:+1: for switching to env. One option, instead of two.

Reply:
[53c0b82](https://github.com/camunda/camunda-modeler/pull/3792/commits/53c0b823234f61d7e41141d48ed18d9a44e22ece)